examples/benchmarks/flops_performance.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Model benchmark example for FLOPs performance.
+
+Commands to run:
+  python3 examples/benchmarks/flops_performance.py
+"""
+
+from superbench.benchmarks import BenchmarkRegistry, Platform
+from superbench.common.utils import logger
+
+if __name__ == '__main__':
+    parameters = '--n 16384 --k 16384 --m 16384 --precision FP32 TF32_TC FP16_TC'
```

Comment:
In example, do we cover all common precision?

Reply:
The most commonly used if FP32, TF32_TC and FP16_TC, what other precision do you suggest? BF16_TC?

Reply:
All precisions

Reply:
Done.